MetaMask/metamask-extension
# Feat/13662 box design token updates

Fixes: #13662
Dependencies: #13657

**TO DO**
- [x] Merge #13657
- [x] Update design-tokens to v`1.3.0`

Explanation: Updating `Box` component to use new design token colors and updating documentation in storybook

Manual testing steps: 
  - Check latest storybook build and view `Box` story
 **OR**
  - run `yarn setup && yarn storybook`
  - Go to `Box` and use background color and border color controls http://localhost:6006/?path=/story/ui-components-ui-box-box-stories-js--default-story&globals=theme:dark
  - Go to `Box / BackgroundColor` story http://localhost:6006/?path=/story/ui-components-ui-box-box-stories-js--background-color&globals=theme:dark use theme changer to view dark theme colors
  - Go to `Box` docs to read added documentation http://localhost:6006/?path=/docs/ui-components-ui-box-box-stories-js--background-color
 
## Screenshots
<img width="1440" alt="Screen Shot 2022-02-23 at 12 19 54 PM" src="https://user-images.githubusercontent.com/8112138/155249581-a68291ec-3cb4-46e5-84bf-8a8763d6af72.png">

Ready for storybook darkmode when [13651](https://github.com/MetaMask/metamask-extension/pull/13651) is merged
<img width="746" alt="Screen Shot 2022-02-23 at 12 31 11 PM" src="https://user-images.githubusercontent.com/8112138/155250288-1ed83745-09a3-47a9-bc2c-b6451580919a.png">

![box](https://user-images.githubusercontent.com/8112138/155249610-cf953878-dfcd-4cb0-9e59-7c977a6cded2.png)
